src/pages/Candidate/index.js

```diff
@@ -0,0 +1,153 @@
+                },
+                {
+                    key: 'Foundation Vote Status',
+                    value: <Tooltip title="Foundation Vote Status">
+                        <>
+                            {candidateInfo.foundationVoteStatus === "supported" ? <LabelText labelType="success">
+                                Supported
+                            </LabelText> : <LabelText labelType="warning">
+                                Rejected
+                            </LabelText>}
+                        </>
+                    </Tooltip>
+                },
+                {
+                    key: 'Last Vote Timestamp',
+                    value: <Tooltip title="Last Vote Timestamp">
+                        <span>{candidateInfo.lastVoteTimestamp}</span>
+                    </Tooltip>
+                },
+                {
+                    key: 'Owner Address',
+                    value: <Tooltip title="Owner Address">
+                        <span>{candidateInfo.ownerAddress}</span>
+                    </Tooltip>
+                },
+                {
+                    key: 'Proposal Fee (EVRs)',
+                    value: <Tooltip title="Proposal Fee (EVRs)">
+                        <span>{candidateInfo.proposalFee}</span>
+                    </Tooltip>
+                },
+                {
+                    key: 'Short Name',
+                    value: <Tooltip title="Short Name">
+                        <span>{candidateInfo.shortName}</span>
+                    </Tooltip>
+                },
+                {
+                    key: 'Status',
+                    value: <Tooltip title="Status">
+                        <>
+                            {candidateInfo.status === "supported" ? <LabelText labelType="success">
+                                Supported
+                            </LabelText> : candidate.status === "elected" ?
+                                <LabelText labelType="primary">
+                                    Elected
+                                </LabelText> : candidate.status === "vetoed" ?
+                                    <LabelText labelType="danger">
+                                        Vetoed
+                                    </LabelText> : candidate.status === "expired" ?
+                                        <LabelText labelType="dark">
+                                            Expired
+                                        </LabelText> : <LabelText labelType="warning">
+                                            Rejected
+                                        </LabelText>}
+                        </>
+                    </Tooltip>
+                },
+                {
+                    key: 'Status Change Timestamp',
+                    value: <Tooltip title="Status Change Timestamp">
+                        <span>{candidateInfo.statusChangeTimestamp}</span>
+                    </Tooltip>
+                },
```

Comment:
New hook candidate type will have governorHookHash, registryHookHash, heartbeatHookHash properties. Let's show them if they are presenting in the object